README.md

```diff
@@ -683,13 +683,13 @@ Offline Installations   | ✓     | x   | x   | x    |
 
 Tool   | Version |
 ---    | ---     |
-Go     | 1.15.12 |
+go     | 1.16.5 |
 npm    | 6.14.13 |
 node   | 14.17.0 |
-pip    | 20.2.2  |
-Python | 3.9.5   |
+pip    | 21.0.1  |
+python | 3.9.6   |
 git    | 2.31.1  |
-Yarn   | 1.x     |
+yarn   | 1.x     |
```

Comment:
I saw that yarn is not really installed in the worker pods, am I right? In this case, it is not possible to write a specific version for Yarn here. 

Do you think we should add a notice that says this version is related to the ability of yarn.lock processing?

```
* The Yarn runtime is not used by Cachito. The processing of yarn.lock files is made by PYarn, which is compatible with any 1.x file.  
```

Reply:
> Do you think we should add a notice that says this version is related to the ability of yarn.lock processing?

Yes, there could be note like (via PYarn).

Also in the first column is a tool name, but not executable. So some names could be capitalized or written in a free form.

Reply:
Yup, I was just kind of bothered that some of them were capitalized and some not (nevermind me, I can focus on unimportant things at times :sweat_smile:). I just changed them all to be capitalized, it seems to be the most correct form.